pkgs/emacs/wrapper.nix

```diff
@@ -69,6 +71,36 @@
     );
     executablePackages = map (pkg: "${pkg}/bin") executablePackages;
   });
+
+  macosAppLauncher = writeShellScript "emacs-launcher" ''
+    SCRIPT_DIR="$( cd "$( dirname "''${BASH_SOURCE[0]}" )" && pwd )"
+    APP_CONTENTS="$(dirname "$SCRIPT_DIR")"
+    APP_RESOURCES="$APP_CONTENTS/Resources"
+
+    # Use a login shell to source the Nix environment
+    # This ensures we get the same environment
+    exec -l "$SHELL" -c '
+      if [ -d "'$APP_RESOURCES'/site-lisp" ]; then
```

Comment:
Please use double brackets in all places.

Reply:
Whoops! I did do this in an earlier change, but looks like I lost the changes in a rebase. One sec, I'll fix these :)